services/platform/docker-entrypoint.sh

```
@@ -2,701 +2,283 @@
-  local timeout="${2:-30}"
-  local service_name="${3:-Service}"
-  local allow_timeout="${4:-false}"
-
-  echo "⏳ Waiting for ${service_name} at ${url}..."
-
+  local url="$1" timeout="${2:-60}" name="${3:-Service}" allow_timeout="${4:-false}"
+  log_info "Waiting for ${name} at ${url}..."
   local counter=0
   until curl -sf "$url" > /dev/null 2>&1; do
     counter=$((counter + 1))
     if [ $counter -gt $timeout ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)